.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question?
+    url: https://www.biodrop.io/docs/faqs
+    about: If your question is not covered by the FAQs, please ask your question in our GitHub Discussions.
```

Comment:
One last suggestion and GTG, can we have our discord over rather than this? Thanks a lot

Reply:
How do I link it? With an invite link? My invite may expire after 7 days.

Reply:
https://discord.eddiehub.org.